.env.j2

```diff
@@ -1,9 +1,5 @@
-# Flask server configuration.
 FLASK_APP=todo_app/app
 FLASK_ENV=production
-
-# Change the following values for local development.
 SECRET_KEY= {{ secret_key }}
-TRELLO_KEY= {{ trello_key }}
-TRELLO_TOKEN= {{ trello_token }}
-TRELLO_BOARD_ID= {{ trello_board_id }}
+DB_NAME={{ db_name }}
+MONGODB_CONNECTION_STRING= {{ mongodb_connection_string }}
```

Comment:
Nice work - don't feel obliged to maintain this/any other Ansible stuff going forward but it's certainly what we'd want in reality!